### ISSUE TITLE
py-mini-racer on Windows

### ISSUE THREAD

**Issue by Shildifreak** [sqreen/PyMiniRacer#101]
Hi,
I wanted to install py-mini-racer on windows, but although they should exist according to the readme I couldn't find any binary files anywhere. Did I miss something? Do they just not exist yet?

Anyway, so the pip wheel did contain the sources... and I tried to compile that...
which after 6 hours or so of googling compiler settings and rewriting code to not use pthreads, have the right paths, figuring out static vs dynamic linking etc, did eventually work out.
Just for me to find out this morning, that all this work had already been done in the github version which just isn't on pip yet. (Feeling pretty stupid to not have checked that first.)

Ok, dropping the plan of forking the repo and polishing up my rather dirty fixes and back to compiling. Actually it went pretty smooth this time, figured out that missing build_ext option was a problem of not updating setuptools, mklink seems to require elevated privilege in recent windows 10 version, but apart from that it just worked.

TLDR;
windows binaries or at least an updated source release would be great
(Oh and thanks to everyone who works/worked on this project - it's super usefull :)

**Comment by lasoke** [comment]
Hi,

We have the same issue.
First command that we have to run is not working, it doesn't give me any trust in the solution.
I really hope this solution is still in beta and the quality of the solution will increase before the production release.

**Comment by nizox** [comment]
A binary version of the module was uploaded to PyPi for the 0.2 release. You can find it [here](https://pypi.org/project/py-mini-racer/0.2.0/#files). Sorry you had to build it yourself before!
